### PR TITLE
Updated GitHub repository paths

### PR DESCRIPTION
**Pull request by cwarnermm**
Need Engineering guidance to resolve the following outdated paths:
- ``https://github.com/mattermost/mattermost-server/blob/master/build/PRIVACY_POLICY.md``
- ``https://github.com/mattermost/mattermost-server/blob/master/model/feature_flags.go#L11``
- ``https://github.com/mattermost/mattermost-server/blob/master/model/config.go``
- ``https://github.com/mattermost/mattermost-server/model``
- ``https://github.com/mattermost/mattermost-server/blob/master/services/telemetry/telemetry.go#L1406``
- ``https://github.com/mattermost/mattermost-server/blob/master/scripts/ldap-check.sh``

@sbishel - Can a member of your team help identify the new monorepo equivalent-paths needed please?

### CODE REVIEW

**Comment by sbishel** [comment]
Essentially replace "mattermost-server/blob/master" with "mattermost/blob/master/server".

https://github.com/mattermost/mattermost-server/blob/master/build/PRIVACY_POLICY.md
to
https://github.com/mattermost/mattermost/blob/master/server/build/PRIVACY_POLICY.md


https://github.com/mattermost/mattermost-server/model
to
https://github.com/mattermost/mattermost/server/public/model

**Comment by cwarnermm** [comment]
@sbishel - I've attempted to do so; however, many of the resulting URLs lead to 404 errors. For example: https://github.com/mattermost/mattermost/server/public/model.

I suspect that perhaps the files I've listed in the summary description of this PR have moved more substantially.

**Comment by sbishel** [comment]
So, I guess, the bottom one needs "tree" added as well, but so would have the original path. 

https://github.com/mattermost/mattermost/tree/master/server/public/model
https://github.com/mattermost/mattermost/blob/master/server/public/model/feature_flags.go#L11
https://github.com/mattermost/mattermost/blob/master/server/public/model/config.go
https://github.com/mattermost/mattermost/blob/master/server/platform/services/telemetry/telemetry.go
https://github.com/mattermost/mattermost/blob/master/server/scripts/ldap-check.sh

File: source/conf.py [hunk 1]
```diff
@@ -2572,9 +2572,9 @@ def setup(_: Sphinx):
 # built documents.
 #
 # The short X.Y version.
-# version = '8.1'
+# version = '9.0'
 # The full version, including alpha/beta/rc tags.
-# release = '8.1'
```
[the remark this answers is not in the document]
Comment (cwarnermm) [reply source/conf.py]: Great catch, @amyblais! I selected the wrong branch by mistake. Corrected now.

**Review by amyblais — APPROVED** [review]

**Pull request merged**